shared/src/formationData.ts

```diff
@@ -295,7 +295,7 @@ export const createEmptyFormationFormData = (): FormationFormData => {
     contactPhoneNumber: "",
     foreignDateOfFormation: undefined,
     foreignStateOfFormation: undefined,
-    willPracticeLaw: undefined,
+    willPracticeLaw: false,
```

Comment:
This is also subject to change as the behavior for whether the radio button should have no selection or no by default was unclear from product.

Reply:
But its a one line change that I don't think interferes with the review.